admin/API_CN.md

```
@@ -0,0 +1,395 @@
+平台提供了一整套 API 来帮助用户管理 API 网关的资源映射、插件配置以及请求处理。文档中的示例涵盖了常见的请求与响应格式，并介绍了如何使用
+Postman 进行接口测试。
+
+无论是创建新资源、修改现有配置，还是管理插件组，本文档都提供了清晰的步骤和必要的 API 细节，方便开发者快速上手并进行集成。
+
+更多的 API 具体介绍请参考 [Swagger 文档](./doc/swagger.json)
+
+## 返回值说明
+
+* **code**：
+
+    * `10001`: 成功
```

markdownlint-cli2 (0.18.1)

Unordered list indentation
Expected: 2; Actual: 4

(MD007, ul-indent)